backend/src/main/java/quarano/department/web/TrackedCaseContactSummary.java

```diff
@@ -87,6 +87,11 @@ public Boolean getIsSenior() {
 		return contactPerson.getIsSenior();
 	}
 
+	public String getRemark() {
+
+		return contactPerson.getRemark();
```

Comment:
Just as a note: We write simple methods with only a single-line statement compactly without empty lines.
https://quarano.de/internal-dev/api/docs/index.html#_blank_lines